.github/workflows/go.yml

```diff
@@ -12,7 +12,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.14', '1.13', '1.12', '1.11' ]
+        go:
+        - 1.11
+        - 1.12
+        - 1.13
+        - 1.14
+        - 1.15
+        - 1.16
+        - 1.17
```

Comment:
If running on every version is a bit too much. Maybe we can just support only the last 3 (or 4) versions. ^^